app/scripts/modules/core/src/manifest/stage/JobManifestPodLogs.tsx

```diff
@@ -61,9 +63,14 @@ export class JobManifestPodLogs extends React.Component<IJobManifestPodLogsProps
     const region = this.resourceRegion();
     InstanceReader.getConsoleOutput(account, region, this.podName(), 'kubernetes')
       .then((response: IInstanceConsoleOutput) => {
+        const containerLogs = response.output as IInstanceMultiOutputLog[];
+        containerLogs.forEach((log: IInstanceMultiOutputLog) => {
+          log.formattedOutput = DOMPurify.sanitize(new AnsiUp().ansi_to_html(log.output));
```

Comment:
do we need to create a new `ansiUp` object on every click?

Reply:
nope, we don't. Will fix